package.json

```diff
@@ -35,6 +37,7 @@
     "gulp-less": "~1.2.3",
     "gulp-replace": "~0.3.0",
     "protractor": "~0.24.0",
-    "http-server": "~0.6.1"
+    "http-server": "~0.6.1",
+    "jpm": "^1.0.0"
```

Comment:
Mozilla mentions [here](https://github.com/mozilla/jpm) that jpm `Currently only works with Firefox 38+`, which is [being released in May](https://developer.mozilla.org/en-US/Firefox/Releases/38). Do you know if there is any way to to automate building of a Firefox extension that would allow `ng-inspector` to support older versions?